test-harness/index.ts

```diff
@@ -36,12 +36,11 @@ describe('harness', () => {
     afterAll(() => tmpFileHandle.removeCallback(undefined, undefined, undefined, undefined));
 
     test(parsed.test, done => {
-      expect.hasAssertions()
       const [command, ...args] = parsed.command.split(' ').map(t => t.trim());
       const serverArgs = [...parsed.server.split(' ').map(t => t.trim()), tmpFileHandle.name];
 
       prismMockProcessHandle = cp.spawn(
-        path.join(__dirname, '../cli-binaries/prism-cli-linux'),
+        path.join(__dirname, '../cli-binaries/prism-cli'),
```

Comment:
Am I reading this all right, that we call the windows one `cli-binaries/prism-cli` so this works, but then we build linux and osx at the same time with `npx pkg --out-path ./cli-binaries --targets linux,macos ./packages/cli/`? Is that not gonna be trouble for CircleCI?

Reply:
Very good spot here. Let me see how I can fix this.

Reply:
@philsturgeon 
Actually this is ok as it is. The multi binary creation `linux,macos` is executed in the "publish" step, when the harness has already ran with linux.

Here you can see the harness uses a binary called `prism-cli`
https://github.com/stoplightio/prism/blob/master/.circleci/config.yml#L39-L57

While the public process is then later creating binary named per platform

https://github.com/stoplightio/prism/blob/master/.circleci/config.yml#L59-L82

These are two separate steps and they do not collide. Does this clarify?